src/envo/envo-edit.owl

```diff
@@ -28119,6 +28142,155 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> <http://purl.obolibrar
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000045> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000045> "brine rejection process"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000045> <http://purl.obolibrary.org/obo/ENVO_03000043>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000045> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_00002149>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000045> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002234> <http://purl.obolibrary.org/obo/ENVO_00003044>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000046> (frazil)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://nsidc.org/cryosphere/glossary/term/frazil") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000046> "Small needle-like ice crystals consisting of nearly pure fresh water, which are suspended in water.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000046> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000046> "frazil"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000046> <http://purl.obolibrary.org/obo/ENVO_01000846>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000047> (frazil formation)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://nsidc.org/cryosphere/glossary/term/frazil") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000047> "A material accumulation process in which frazil ice crystals merge, forming thin sheets of ice.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000047> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000047> "frazil formation"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000047> <http://purl.obolibrary.org/obo/ENVO_03000009>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000047> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_03000046>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000047> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002234> <http://purl.obolibrary.org/obo/ENVO_00000132>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000048> (thaw hole)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://nsidc.org/cryosphere/glossary-terms/sea-ice") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000048> "vertical hole in floating ice formed when a puddle melts through to the underlying water.")
```

Comment:
@pbuttigieg but is there any ice that wouldn't float? isn't that confusing? i would think it's just a matter of compression?